crypto/ec/ec_local.h

```diff
@@ -213,6 +213,7 @@ struct ec_group_st {
     BIGNUM *order, *cofactor;
     int curve_name;             /* optional NID for named curve */
     int asn1_flag;              /* flag to control the asn1 encoding */
+    int from_explicit_encoding; /* set if decoded from explicit encoding */
```

Comment:
I know it's longer and we usually hate long names, but it really should be `from_explicit_parameters_encoding`.

I would understand if for brevity you prefer to flip it to `from_named_curve_encoding`, but explicit encoding really risks creating more confusion.

Reply:
In the end I decided to use decoded_from_explicit_params @romen is that sufficiently non-confusing?